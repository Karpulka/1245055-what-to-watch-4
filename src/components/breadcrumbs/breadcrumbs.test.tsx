import * as React from "react";
import renderer from "react-test-renderer";
import Breadcrumbs from "./breadcrumbs";
import {StaticRouter} from "react-router-dom";

it(`Render breadcrumbs`, () => {
  const tree = renderer
    .create(<StaticRouter>
      <Breadcrumbs id={1} title={`Test title`}/>
    </StaticRouter>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
