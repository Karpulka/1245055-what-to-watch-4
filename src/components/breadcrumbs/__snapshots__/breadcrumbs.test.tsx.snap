// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render breadcrumbs 1`] = `
<nav
  className="breadcrumbs"
>
  <ul
    className="breadcrumbs__list"
  >
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
        href="/films/1"
        onClick={[Function]}
      >
        Test title
      </a>
    </li>
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
        href="/"
        onClick={[Function]}
      >
        Add review
      </a>
    </li>
  </ul>
</nav>
`;
